docs/user-guide/dhcp.md

```diff
@@ -0,0 +1,346 @@
+# DHCP
+
+The Fabric includes an integrated DHCP server that dinamically assigns IP addresses to hosts connected to VPC subnets. The DHCP server supports dynamic IP allocation, static leases, and various configuration options.
+
+## Overview
+
+The DHCP server is configured per VPC subnet and provides:
+
+- Dynamic IP allocation from a configurable range
+- Static leases for specific MAC addresses
+- DHCP options including DNS servers, NTP servers, PXE boot, MTU, and custom routes
+- DHCP relay support for third-party DHCP servers
+
+## Basic Configuration
+
+### Enabling DHCP
+
+To enable DHCP for a VPC subnet, set `dhcp.enable: true` in the subnet configuration:
+
+```yaml title="vpc-with-dhcp.yaml"
+apiVersion: vpc.githedgehog.com/v1beta1
+kind: VPC
+metadata:
+  name: vpc-1
+  namespace: default
+spec:
+  subnets:
+    default:
```

Comment:
Could you add a comment or annotation to mention that this is the name of the subnet and can be changed as the user desires?